react/week3/week3-react/src/index.css

```
@@ -0,0 +1,147 @@
+/*body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+    monospace;
+}*/
+*{
+  box-sizing:border-box;
+}
+body{
+  margin : 0;
+  padding: 0;
+  font-family: sans-serif;
+  /* background: black; */
+  color: palegreen;
+  
+}
+.button {
+  background-color: lightgoldenrodyellow; /* Green */
+  border: none;
+  color: goldenrod;
+  padding: 15px 32px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  cursor: pointer;
+}
+.button:hover {
+  background-color: rgb(50, 169, 220);
+}
+.text-center{
+  text-align:center;
+  text-decoration: underline;
+  color:palevioletred;
+  font-size: 25px;
+  font-style:inherit ;
+  margin-top: 0px;
+  padding: 10px;
+  
+}
+.container{
+  width:100%;
+  margin: 0% auto;
+  color: red;
+  background-color: orange;
+  background: url("components/Images/Image1.jpg");
+  height:550px;
+
+}
+.ListItems {
```

```suggestion
.list-items {
```
CSS classes should be in kebab-case